proto/compiler.bzl

```diff
@@ -117,13 +117,14 @@ def go_proto_compile(go, compiler, protos, imports, importpath):
     args.add_all(imports, before_each = "-import")
     args.add_all(proto_paths.keys())
     args.use_param_file("-param=%s")
+    data = [file for target in compiler.internal.data for file in target.files.to_list()]
```

Comment:
This is problematic as it flattens a `depset` with `to_list()`, which should always be avoided. 

Instead, could you collect the individual `target.files` in a list and pass them to the `transitive` parameter of `depset` below?

Reply:
@fmeum Thanks for the tip. Updated as you described.